.github/workflows/zip.yml

```diff
@@ -13,6 +13,13 @@ on:
     # Run every day at midnight(PST) - cron uses UTC times
     - cron:  '0 8 * * *'
 
+  workflow_dispatch:
+    inputs:
+      custom_spec_repos:
+        description: 'Custom Podspec repos'
+        required: true
+        default: 'https://cpdc-internal.git.corp.google.com/firebase.git'
```

Comment:
I don't think GHA has access to `https://cpdc-internal.git.corp.google.com`. I believe we need the whole staging process to SpecsStaging or another public repo.

Reply:
Yeah, I would prefer moving release staging to GitHub to simplify GHA access control. I think we can address it in following PRs. For now I would like to test the manual trigger in general.